pkg/user/signer.go

```diff
@@ -23,9 +20,25 @@ import (
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	abci "github.com/tendermint/tendermint/abci/types"
 	"google.golang.org/grpc"
+
+	"github.com/celestiaorg/celestia-app/v2/app/encoding"
+	apperrors "github.com/celestiaorg/celestia-app/v2/app/errors"
+	blobtypes "github.com/celestiaorg/celestia-app/v2/x/blob/types"
 )
 
-const DefaultPollTime = 3 * time.Second
+const (
+	DefaultPollTime              = 3 * time.Second
+	DefaultGasMultiplier float64 = 1.1
```

Comment:
[not blocking]
Wouldn't this be better if it's configurable and exposed to the user someway? At some point, 1.1 might not be enough too, if I understand correctly

Reply:
I'm not sure that the user should be aware of any multipliers that we have. The current multiplier covers the case when our Estimation is not perfect(in case of additional options). I also think we'll never face the issue when we have to increase the `DefaultMultiplier`.  Signer has an option that allows to configure the multiplier if it's required and imo it's enough.

Reply:
Yeah this value is configurable as a `signer.Option`